code/modules/mob/mob.dm

```diff
@@ -1334,7 +1334,7 @@ mob/proc/yank_out_object()
 		to_chat(host, "<span class='info'>You are now a mouse. Try to avoid interaction with players, and do not give hints away that you are more than a simple rodent.</span>")
 
 /mob/proc/assess_threat() //For sec bot threat assessment
-	return
+	return 5
```

Comment:
Won't this line mean that most sec bots will regard all mobs as dangerous by default?